react/features/external-api/middleware.js

```diff
@@ -88,11 +88,10 @@ MiddlewareRegistry.register(store => next => action => {
     case CONFERENCE_JOINED: {
         const state = store.getState();
         const { defaultLocalDisplayName } = state['features/base/config'];
-        const { room } = state['features/base/conference'];
         const { loadableAvatarUrl, name, id } = getLocalParticipant(state);
 
         APP.API.notifyConferenceJoined(
-            room,
+            APP.conference.roomName,
```

Comment:
Could you please add a comment why do we use `APP.conference.roomName` instead of redux? I guess we would want to probably fix the value in redux some day...